fixtures/flutter_app/android/local.properties

```diff
@@ -0,0 +1,5 @@
+sdk.dir=/Users/jacobr/Library/Android/sdk
```

Comment:
Do we need this file?

Reply:
good catch. this directory was actually inserted. Not sure why my client had those spurious files. The fixtures are now all under the separate devtools_testing package.